mods/cnc/maps/cnc64gdi01/map.yaml

```diff
@@ -731,6 +731,23 @@ Actors:
 Smudges:
 
 Rules:
+	^Palettes:
```

Comment:
The definition depends on the players defined in the map, so that's not really feasible.